ironfish-cli/src/commands/testnet.ts

```diff
@@ -124,6 +133,14 @@ export default class Testnet extends IronfishCommand {
       this.log('')
     }
 
+    if (!flags.confirm && updateTelemetry) {
+      updateTelemetry = await CliUx.ux.confirm(
+        'Do you want to help improve Ironfish by enabling Telemetry? (y)es / (n)o',
```

Comment:
Branding nit: "Ironfish" -> "Iron Fish"

Reply:
I think Iron Fish is the company, and the product is ironfish, but I'll change this to refer to the company.

Reply:
Just for posterity's sake, It seems that we also refer to the product as Iron Fish as well; some examples from the website:
> Iron Fish is a Layer 1 blockchain...

and

> Iron Fish is built to be intuitive for both developers and users alike.